### PR TITLE
[Bugfix] Label expression in with clause has default alias: its label name

### PR DESCRIPTION
**Pull request by jievince**
fix https://github.com/vesoft-inc/nebula-graph/issues/663, fix https://github.com/vesoft-inc/nebula-graph/issues/624

### CODE REVIEW

File: src/validator/MatchValidator.cpp [hunk 1]
```diff
@@ -450,6 +450,9 @@ Status MatchValidator::validateWith(const WithClause *with,
     std::vector<const Expression *> exprs;
     exprs.reserve(with->columns()->size());
     for (auto *col : with->columns()->columns()) {
+        if (col->expr()->kind() == Expression::Kind::kLabel) {
+            col->setAlias(new std::string(col->expr()->toString()));
+        }
         if (col->alias() == nullptr) {
```
Comment (Shylock-Hg) [inline src/validator/MatchValidator.cpp:456]: I think you could remove this check.

Comment (jievince) [reply src/validator/MatchValidator.cpp:456]: Just add an alias for `Label Expression` implicitly. So this check is necessary

**Review by yixinglu — APPROVED** [review]

**Review by bright-starry-sky — APPROVED** [review]
LGTM

**Review by Shylock-Hg — APPROVED** [review]
LGTM

**Pull request merged**